index.html

```diff
@@ -28,7 +28,7 @@ <h1>Dr.Norman Borlaug</h1>
       <figure id="img-div" class="">
         <img src="./image-asset.jpeg" alt="" id="image" class="img-fluid" />
         <figcaption id="img-caption">
-          <span class="fw-bold"
+          <span class="fw-bold" id="fw-bold"
             >Dr. Norman Borlaug, third from the left, trains biologists in
```

Comment:
Please use semantic html like <article>instead of span for this code.

Reply:
> Please use semantic html like
> 
> instead of span for this code.

- [ ] Kindly always remember to give a reason for your suggestion, while you did it for other comments, I would implore you to give a reason here too so that the student won't be making your requested changes in the dark, the student will reason with you and also learn from your wealth of experience.